book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  has_many :rentals
+
+  attr_accessor :title, :author
```

Comment:
- [ ] Great job working on the association so far but I think it would be great if you initialized the rentals here so kindly add it here in your constructor 🔥✅💯 here's a link for a [lesson](Link to the example about constructors in a previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to understand better ✨

![image](https://github.com/jkanyi-web/OOP_School_Library/assets/58553711/dbaf4e2f-c5f3-479a-95ca-124aa17603ad)